### DISCUSSION TITLE
Welcome to unotes Discussions!

### DISCUSSION THREAD

**Discussion by ryanmcalister** [ryanmcalister/unotes#99] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kiyoka** [comment]
Hello.
I would like to put in image related improvements to Unotes.
Most of the documentation I create is procedural documentation.
Below is my general workflow.
- Paste a screen capture into a markdown document.
- Add text to explain it.
- Add a mark to the pasted image and save it overwrite.

I was recently a contributor to Mark Text, but Mark Text development has stopped.
I would like to implement similar functionality in Unotes.

**Reply by ryanmcalister** [reply]
Thanks for adding features to Unotes. I welcome the help!
